src/App.js

```diff
@@ -25,6 +26,7 @@ function App(props) {
             <Switch>
               <Route exact path="/" component={Landing}/>
               <Route exact path="/expired" component={Expired}/>
+              <Route exact path="/confirmation" component={Confirmation}/>
               <Route path="/:pollUrl">
                 {auth.isAuthenticated ? <Poll/> : <Dummy/>}
               </Route>
```

Comment:
```suggestion
              <Route exact path="/:pollUrl">
                {auth.isAuthenticated ? <Poll/> : <Dummy/>}
              </Route>
               <Route exact path="/:pollUrl/confirmation" component={Confirmation}/>
```
Zapomniałem napisać tego w tasku ale confirmation powinien odnosic sie do tej danej ankiety czyli url powinien być np. `/cute-little-beaver/confirmation`